buildSrc/build.gradle.kts

```diff
@@ -26,23 +26,23 @@ object Versions {
     const val detekt = "1.20.0-RC1"
     const val dokka = "1.6.10"
     const val kotlin = "1.6.20"
-    const val ktlint = "10.2.1"
```

Comment:
For clarity, we were actually using this version, `10.2.1` of the ktlint gradle plugin -- https://github.com/JLLeitschuh/ktlint-gradle/tree/v10.2.1.